src/driver/driver.rs

```diff
@@ -8,6 +8,9 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#![allow(unknown_features)]
```

Comment:
This may be best expressed as `#[cfg_attr(stage0, allow(unknown_features))]` as it's something easy to grep for to remove afterwards (grepping for "stage0")